k8s/upgrades/0.9.0-1.0.0/README.md

```diff
@@ -29,6 +29,25 @@ $ cd openebs/k8s/upgrades/0.9.0-1.0.0/
 ## Step 1: Prerequisites
 
 *All steps described in this document need to be performed on the Kubernetes master or from a machine that has access to Kubernetes master*
+- If OpenEBS has been deployed using openebs helm charts, it has to be in chart version `0.9.2` . Run `helm list` to verify the chart version.
+   If not, we have to update openebs chart version using below commands.
+
+    - Firstly we have to delete the `admission-server` secret, which will be deployed again once we upgrade charts to `0.9.2` version using below command:
+      ```sh
+      $ kubectl delete secret admission-server-certs -n openebs
+      ```
+
+    - Upgrade OpenEBS chart version to 0.9.2 using below command:
+      ```sh
+      $ helm upgrade <release-name> stable/openebs --version 0.9.2
```

Comment:
prior to this .. let us include `helm repo update`

Reply:
Done